src/components/Dialog/index.tsx

```diff
@@ -239,18 +244,17 @@ type DialogBodyProps = {
   children: ReactNode
   title?: string
   back?: string
+  height?: string
```

Comment:
I don't really like introducing a height prop for this. It makes the logic a bit hidden. Also, it leaves room for a lot of inconsistency. If you pass a height that is larger than the actual content, you have a lot of spacing left without any content, which looks weird as well.

<img width="1666" alt="Screenshot 2022-11-30 at 11 37 20" src="https://user-images.githubusercontent.com/14276144/204775184-d6990ca3-8ddd-4fb1-a691-7f857a53e294.png">

Reply:
Adding fixed heights could be a good way to go, so e.g. allowing users to pass a S M or L prop or something?

One concern I have is that with scrollable content, we want to convey to users that they can scroll. To do that, the dialog needs to be cut off at precise points so you see that there is content 'down', e.g. like below (where Mark also points it out).

If we only have a couple of predetermined heights, we don't give ourself a lot of control to decide where to cut the dialog off as content will differ.

<img width="614" alt="image" src="https://user-images.githubusercontent.com/49148610/205008086-fa1ec3ec-f39c-4802-8359-ab8e75155896.png">

Reply:
Also, if you do pass a height that is larger than the content then it'll indeed look funny, but I don't think we're planning to pass dynamic heights, right? And when else would you give it a larger height than the content.